tests/test_behavior/test_audio_interface.py

```diff
@@ -258,3 +255,32 @@ class AudioTestNWBConverter(NWBConverter):
                 # Try to read the data
                 acoustic_series = nwbfile.stimulus[audio_name].data[:]
                 assert len(acoustic_series) > 0
+
+    def test_multiple_dots_in_filename(self):
+        """Test that AudioInterface works with WAV files that have multiple dots in filename."""
+        with tempfile.TemporaryDirectory() as temp_dir:
+            # Create a WAV file with multiple dots in the filename
+            file_path = Path(temp_dir) / "test.1.wav"
+            create_24bit_wav_file(file_path)
+
+            # Initialize a fresh AudioInterface with the file
+            interface = AudioInterface(file_paths=[file_path])
+            # Get metadata
+            metadata = interface.get_metadata()
+            metadata["NWBFile"].update(session_start_time=self.session_start_time)
+
+            # Run conversion
+            nwbfile_path = str(self.test_dir / "audio_test.1.nwb")
+            interface.run_conversion(nwbfile_path=nwbfile_path, metadata=metadata, overwrite=True)
```

Comment:
Can we just `add_to_nwbfile` to check that the error is not triggered so we don't write to file in this case? Just to make the tests faster saving somet time.